client/src/components/dashboard/verify/Verify.tsx

```
@@ -0,0 +1,137 @@
+      </Stepper>
+
+      <div className="p-4 text-center ">
+        {activeStep === 0 && (
+          <div>
+            <label className="flex flex-col items-center justify-center border-2 border-dashed border-gray-400 p-4 rounded-lg cursor-pointer hover:border-blue-500">
+              <UploadCloud className="w-10 h-10 text-gray-500 mb-2" />
+              <span className="text-gray-600">Upload Student ID Image</span>
+              <input type="file" className="hidden" onChange={handleFileChange} />
+            </label>
+            {image && <p className="mt-2 text-sm text-green-600">File selected: {image.name}</p>}
+            <button onClick={handleUpload} className="mt-4 px-4 py-2 bg-blue-600 text-white rounded hover:bg-blue-800">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)